src/components/checkBoxSelection/checkBoxSelection.js

```diff
@@ -1,28 +1,52 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import './checkBoxSelection.scss';
+import { connect } from 'react-redux';
+import { ADD_VARIABLE, UPDATE_VARIABLE, DELETE_VARIABLE } from '../../store/actionTypes';
+
 class CheckBoxSelection extends Component {
     constructor(props) {
         super(props);
         this.state = {
-            isClicked: false
-        };
+			 isClicked: false
+		 };
     }
 
     componentWillMount() {
-	 this.setState({ isClicked: this.props.isSelected }); 
-    }
+	   this.setState({ isClicked: this.props.isSelected }); 
```

Comment:
move this into your state declaration constructor instead of using component will mount please

Reply:
i see a thumbs up but no changes in your code